src/plugins/intel_gna/cmake/libGNAConfig.cmake

```diff
@@ -15,40 +15,59 @@ endif ()
 set(libGNA_FOUND TRUE)
 
 set(GNA_KERNEL_LIB_NAME gna CACHE STRING "" FORCE)
-set(GNA_LIBS_LIST
-        "libGNA::API"
-        "libGNA::KERNEL")
 
-set(GNA_LIB_DIR x64 CACHE STRING "" FORCE)
-set(libGNA_INCLUDE_DIRS "${GNA}/include" CACHE STRING "" FORCE)
-set(libGNA_LIBRARIES_BASE_PATH ${GNA}/${GNA_PLATFORM_DIR}/${GNA_LIB_DIR} CACHE STRING "" FORCE)
+if (NOT libGNA_FIND_COMPONENTS)
+    set(libGNA_FIND_COMPONENTS "API;KERNEL")
+endif()
 
-add_library(libGNA::KERNEL SHARED IMPORTED)
-find_library(GNA_KERNEL_LIBRARY
-        ${GNA_KERNEL_LIB_NAME}
-        HINTS
-        ${libGNA_LIBRARIES_BASE_PATH})
-set_target_properties(libGNA::KERNEL PROPERTIES IMPORTED_LOCATION ${GNA_KERNEL_LIBRARY})
+foreach (_gna_component ${libGNA_FIND_COMPONENTS})
+    set(libGNA_${_gna_component}_FOUND FALSE)
+    set(libGNA_FIND_REQUIRED_${_gna_component} TRUE)
+endforeach()
 
-add_library(libGNA::API INTERFACE IMPORTED)
-set_property(TARGET libGNA::API PROPERTY INTERFACE_INCLUDE_DIRECTORIES ${libGNA_INCLUDE_DIRS})
+if(NOT GNA)
```

Comment:
can we rename `GNA` with more meaningful name like `GNA_EXTERNAL_DIR` / ` GNA_PATH` / etc

Reply:
rewrite in other way